api/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "engineStrict": true,
   "engines": {
-    "node": "16.x"
```

Comment:
Why not: >=16.x

Changing to 18.x would make users of node 16 not being able to import the library anymore. >=16.x seems to be a better compromise.